04week/welcome-to-our-site/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Welcome-to-our-site</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="heading">
+        <h1>Welcome to our <strong>site</strong></h1>
+      </div>
+    </div>
+    <div class="titles">
+      <div class="sec" id="fir">
+        <h2>Title</h2>
+        <p>Lorem ipsum dolor <br>
```

Comment:
instead of using `<br>` to control where the words end up, try to use `width` instead on the div or p tag